.github/workflows/st2-docker.yml

```diff
@@ -0,0 +1,42 @@
+name: st2-docker
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: 0 1 * * *
```

Comment:
I think we're missing `pull_request` condition here so the workflow also triggers on PR like this.

Reply:
Ah yes, will fix.

Reply:
Also, the default branch is `master`, I'll fix that too -- I'll leave `main` in case y'all decide to rename someday.

Reply:
...probably won't run till it's landed in `master`, I don't think adding my branch to `on.branches` (since it's coming from a fork, will do the trick).